### DISCUSSION TITLE
Audit categorization / tagging

### DISCUSSION THREAD

**Discussion by plaflamme** [TobikoData/sqlmesh#699] [Ideas]
Some audits may be expensive to run which ~may~ can prevent getting early feedback about problems in other places in a sqlmesh project (say because the expensive audit runs before sqlmesh "reaches" the problem).

Unit tests have a similar problem and some frameworks allow categorizing or tagging them in some way to make them run only conditionally.

Having a similar functionality to `Audit` would be nice. Perhaps something like:

```
Audit(
  name fancy_audit
  tags= [ci, slow]
)
Audit(
  name other_audit
  tags= [ci]
)
```

And then allow specifying which tags to include/exclude on the `cli`

```
sqlmesh plan --audit-tags +ci,-slow # would run other_audit, but not fancy_audit
```

**Comment by tobymao** [comment]
we plan to add tags this quarter. we'll keep this in mind as it should be an easy addition once we have generic tag support for models.

cc @vchan

**Comment by georgesittas** [comment]
Hey @plaflamme, we discussed this internally and concluded that the additional complexity is probably not gonna be high ROI. Models should be viewed as the execution unit, not sure if going more granular than that makes sense.

FWIW this thread was created a long time ago, so maybe there are other tools that can help with this now -- e.g., the model selector.

I'll go ahead and close this to mark the request as out of scope, but happy to continue discussing either here or in Slack.
